engine/src/main/java/org/terasology/rendering/opengl/OpenGLUtil.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2015 MovingBlocks
+ * Copyright 2016 MovingBlocks
```

Comment:
I'd be hesitant to drop the `OpenGL` part just because we might end up in a situation of having multiple `Utilities` classes, having to preface each usage in a file with the full package path. That seems like it would be a little messy :-)

I dunno if there'd be a better more descriptive name that bakes in the exact purpose, somehow splicing together the rendering nature + utility?